python/src/RatioOfUniforms_doc.i.in

```diff
@@ -0,0 +1,487 @@
+%feature("docstring") OT::RatioOfUniforms
+"Ratio of Uniforms simulation algorithm.
+
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Available constructors:
+    RatioOfUniforms(*dist*)
+
+    RatioOfUniforms(*logUnscaledF, rangeF, isScaled*)
+    
+Parameters
+----------
+dist : :class:`~openturns.Distribution`
+    Distribution to sample.
+    
+    With that constructor, the range is the distribution range and *isScaled* = True.
+
+logUnscaledPDF : :class:`~openturns.Function`
+   Function :math:`\log F` such that :math:`f = F\, 1_{I}`  is proportional to the PDF :math:`p` we want to sample.
+
+rangeF : :class:`~openturns.Interval`
+    Interval :math:`I \subset \Rset^\inputDim` outside which :math:`p=0`.
+
+isScaled : bool, optional
+    Flag telling if the input *logUnscaledPDF* is such that :math:`f = F\, 1_{I}` is an actual PDF of a distribution.
+    
+    Default value is False.
+
+Notes
+-----
+This algorithm samples a multivariate distribution whose probability density probability is known up to a multiplicative factor, see [stadlober1990]_ for the unidimensional case and [gobet2016]_ for the multivariate case.
+
+Let :math:`p` be a probability density function of dimension :math:`\inputDim` and
+:math:`f: \Rset^\inputDim \rightarrow \Rset^+` a function such that:
+
+.. math::
+
+    p(\vect{x}) = cf(\vect{x})\quad  \forall \vect{x} \in \Rset^\inputDim
+    
+with :math:`c \in \Rset^+_*`. We can write :math:`f` as:
+
+.. math::
+
+    f(\vect{x}) = F(\vect{x})\, 1_{I}\quad \forall \vect{x}  \in I
+
+with :math:`I \subset \Rset^\inputDim`. It implies that the numerical range of :math:`p` is inluced in :math:`I`. We note:
+
+.. math::
+
+   I = \prod_{i=1}^\inputDim \left[a_i, b_i \right]
+
+If the function :math:`f` is an actual PDF and :math:`I` its numerical range, then :math:`c=1`.
+In the general case, :math:`c` is unknown and assumed to be difficult to compute.
+
+Let :math:`r \in \Rset^+_*`. By default, :math:`r=1`. Let :math:`A_{f,r}` be the domain defined by:
+
+.. math::
+
+    A_{f,r} & = \left\{ (u, v_1, \dots, v_\inputDim) \in \Rset^{\inputDim+1}\, |\, 0 \leq u \leq f\left(\dfrac{v_1}{u^r},
+    \dots, \dfrac{v_\inputDim}{u^r}\right)^{\frac{1}{1+r\inputDim}}\right \}
+
+The Lebesgue measure of :math:`A_{f,r}`  is equal to :math:`\dfrac{1}{c(1+rd)}`.
+
+Let :math:`(U, V_1, \dots, V_\inputDim)` be a random variable uniformly distributed on  :math:`A_{f,r}`.
```

Comment:
a set of random variables ? or a random variable vector ?

Reply:
random vector...